edmarketconnector.sh

```diff
@@ -2,4 +2,5 @@
 
 export TMPDIR="${XDG_RUNTIME_DIR}/app/${FLATPAK_ID}"
 
-cd /app/edmarketconnector; python3 EDMarketConnector.py
+cd /app/edmarketconnector; python3 EDMarketConnector.py 2>&1 | tee "${XDG_DATA_HOME}/EDMarketConnector/logs/EDMarketConnector.log"
+rm "${XDG_DATA_HOME}/EDMarketConnector/logs/EDMarketConnector.log"
```

Comment:
I can't currently think of a better way to keep from an ever growing log file. But, this isn't a good solution as this means the log file is only available when EDMC is running, which could make debugging much harder.